riscv-sbi.adoc

```diff
@@ -1674,6 +1676,334 @@ The possible error codes returned in `sbiret.error` are shown in the
+the platform.
+
+[#figure_attestation_flow]
+.TCB Attestation Flow
+....
+
+                                      ┌──────────────────┐
+                                      │ Reference Value  │
+                                      │     Provider     │
+                                      └────────┬─────────┘
+                                               │
+                                           Reference
+┌───────────┐                               Values
+│ Platform  │                                  │
+├───────────┴──────────┐                       │
+│   ┌────────────┐     │                       ▽
+│   │  Attester  │     │ Layered        ┌────────────┐
+│   │ (e.g. TVM) ├─────┼─Evidence──────▷│  Verifier  │
+│   └────────────┘     │                └──────┬─────┘
+│          △           │                       │
+│          │           │                       │
+│     Evidence for     │                  Attestation
+│       Attester       │                    Results
+│          │           │                       │
+│    ┌─────┴────┐      │                       │
+│    │ Layer 1  │      │                       │
+│    └──────────┘      │                       ▽
+│          △           │             ┌──────────────────┐
+│          │           │             │  Relying Party   │
+│ Evidence for layer 1 │             └──────────────────┘
+│          │           │                       │
+│    ┌─────┴─────┐     │                       │
+│    │  Layer 0  │     │                       │
+│    └───────────┘     │                       │
+└──────────────────────┘                       │
+            △                                  │
+            │                                  │
+            └─────────Secret Injection─────────┘
+
+....
+
+These functions provide the capabilities and features required to implement
+attestation for usages such as confidential computing, measured boot and
+verified boot.
+
+[#get-evidence-function,reftext="Get Evidence"]
+=== Function: Get Evidence (FID #0)
+
+[source, C]
+----
+struct sbiret sbi_attestation_get_evidence(unsigned long cert_request_addr,
+                                           unsigned long cert_request_size
+                                           unsigned long request_data_addr,
+                                           unsigned long evidence_format,
+                                           unsigned long cert_addr,
+                                           unsigned long max_cert_size);
+----
+
+Retrieve a TCB attestation evidence from the underlying SBI implementation.
+Both the constitution and definition of the TCB and its measurement are
+implementation specific.
+
+The caller provides a https://datatracker.ietf.org/doc/html/rfc5280[X.509]
+formatted Certificate Signing Request (CSR) at the `cert_request_addr` physical
+address, which length is `cert_request_size`.
+
+Additionally, the caller provides a 64 bytes data blob at the
```

Comment:
Along lines of Atish's normative text, the "provides" can probably be a "must provide".